src/components/molecules/PasswordChange/index.tsx

```diff
@@ -0,0 +1,129 @@
+//#region > Imports
+//> React
+// Contains all the functionality necessary to define React components
+import React from "react";
+//> MDB
+// "Material Design for Bootstrap" is a great UI design framework
+import {
+  MDBContainer,
+  MDBRow,
+  MDBCol,
+  MDBBtn,
+  MDBCard,
+  MDBCardBody,
+  MDBInput,
+} from "mdbreact";
+//#endregion
+
+//#region > Components
+class PasswordChangeForm extends React.Component<
+  {
+    username: string | undefined;
+    onSubmit: (newPassword: string) => void;
+  },
+  {}
+> {
+  state = {
+    password: "",
+    confirmPassword: "",
+  };
+
+  validPassword = () => {
+    return (
+      this.state.password !== "" &&
+      this.state.confirmPassword !== "" &&
+      this.state.password === this.state.confirmPassword
+    );
+  };
+
+  submitHandler = (event: any) => {
+    event.preventDefault();
+    event.target.className += " was-validated";
+
+    if (this.validPassword()) {
+      this.props.onSubmit(this.state.password);
+    }
+  };
+
+  changeHandler = (event: any) => {
+    this.setState({ [event.target.name]: event.target.value });
+  };
+
+  render() {
+    return (
+      <div>
+        <form
+          className="needs-validation"
+          onSubmit={this.submitHandler}
+          noValidate
+        >
+          <MDBContainer>
+            <MDBRow className="justify-content-center">
+              <MDBCol md="6">
+                <MDBCard>
+                  <MDBCardBody className="mx-4 mt-4">
+                    <p className="font-small grey-text d-flex justify-content-center">
+                      Enter a new password for the user
+                      <p className="dark-grey-text font-weight-bold ml-1">
+                        {this.props.username}
+                      </p>
+                      .
```

Comment:
Why is there a dot?

Reply:
Enter a new password for the user **pinterid** _._ <-- dot